app/src/main/java/com/teamwss/websoso/ui/main/myPage/myActivity/model/ActivitiesModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.teamwss.websoso.ui.main.myPage.myActivity.model
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+data class ActivitiesModel(
+    val isLoadable: Boolean,
+    val feeds: List<ActivityModel>,
```

Comment:
a: 는 어떤가욥? 변수명은 피드를 모아놓은 곳 같은데, 타입을 봤을 땐 활동모델 list 라 일치성을 주면 어떨까 싶어서유!
```suggestion
    val activities: List<ActivityModel>,
```

Reply:
넵 좋아요!